Common/src/Stream/Worker/TaskHandler.cs

```diff
@@ -104,54 +143,21 @@ public IList<string> ExpectedResults
   public async Task CreateTasksAsync(IEnumerable<TaskRequest> tasks,
                                      TaskOptions?             taskOptions = null)
   {
-    await semaphore_.WaitAsync(cancellationToken_)
-                    .ConfigureAwait(false);
+    using var counter = counter_.GetCounter();
+    var       stream  = client_!.CreateTask();
 
-    var requestId = $"R#{messageCounter_++}";
-
-    try
+    foreach (var createLargeTaskRequest in tasks.ToRequestStream(taskOptions,
+                                                                 Configuration.DataChunkMaxSize))
     {
-
-      foreach (var createLargeTaskRequest in tasks.ToRequestStream(taskOptions,
-                                                                   Configuration.DataChunkMaxSize))
-      {
-        await responseStream_.WriteAsync(new ProcessReply
-                                         {
-                                           RequestId       = requestId,
-                                           CreateLargeTask = createLargeTaskRequest,
-                                         },
-                                         CancellationToken.None)
-                             .ConfigureAwait(false);
-      }
-
-      if (!await requestStream_.MoveNext(cancellationToken_)
-                               .ConfigureAwait(false))
-      {
-        throw new InvalidOperationException("Request stream ended unexpectedly.");
-      }
-
-
-      var current = requestStream_.Current;
-
-      if (current.TypeCase != ProcessRequest.TypeOneofCase.CreateTask)
-      {
-        throw new InvalidOperationException("Expected a CreateTask answer.");
-      }
-
-      if (current.CreateTask.ReplyId != requestId)
-      {
-        throw new InvalidOperationException($"Expected reply for request {requestId}");
-      }
-
-      var reply = current.CreateTask.Reply;
-      if (reply.DataCase == CreateTaskReply.DataOneofCase.NonSuccessfullIds)
-      {
-        throw new AggregateException(reply.NonSuccessfullIds.Ids.Select(s => new InvalidOperationException($"Could not create task it id={s}")));
-      }
+      await stream.RequestStream.WriteAsync(createLargeTaskRequest,
+                                            CancellationToken.None)
+                  .ConfigureAwait(false);
     }
-    finally
+
+    var reply = await stream.ResponseAsync.ConfigureAwait(false);
```

Comment:
`.configureAwait(false)` is not on its own line.
/s

Reply:
Appropriatly formatted according to the given rules